app/pages/lab-pages-editor/components/TasksPage.jsx

```diff
@@ -0,0 +1,99 @@
+/* eslint-disable no-console */
+/* eslint-disable react/react-in-jsx-scope */
+/* eslint-disable react/require-default-props */
+/* eslint-disable radix */
+
+import { useWorkflowContext } from '../context.js';
+import strings from '../strings.json'; // TODO: move all text into strings
+
+import GripIcon from '../icons/GripIcon.jsx';
+
+export default function TasksPage() {
+  const { workflow, update } = useWorkflowContext();
+  const isActive = true; // TODO
+
+  function placeholderEventHandler() {
+    console.log('+++ TODO');
+  }
+
+  console.log('+++ workflow: ', workflow);
+
+  if (!workflow) return null;
+
+  return (
+    <div className="tasks-page">
+      <div className="workflow-title flex-row">
+        <h2 className="flex-item">{workflow?.display_name}</h2>
+        <span className="workflow-id">{`#${workflow?.id}`}</span>
+        {(isActive) ? <span className="status-active">Active</span> : <span className="status-inactive">Inactive</span>}
+      </div>
+      <section>
+        <h3>Tasks</h3>
+        <div className="flex-row">
+          <button
+            className="flex-item big primary"
+            onClick={placeholderEventHandler}
+            type="button"
+          >
+            Add a new Task +
+          </button>
+          {/* Dev observation: the <select> should have some label to indicate it's for choosing the starting task. */}
+          <select
+            aria-label="Choose starting test"
+            className="flex-item"
+          >
+            <option disabled={true}>Choose starting test</option>
+          </select>
+        </div>
+        <ul className="steps-list">
+          {/* WARNING: this should be workflow.steps */}
+          {Object.entries(workflow.tasks).map(([key, val]) => (
+            <StepItem
+              task={val}
+              taskKey={key}
+            />
+          ))}
+        </ul>
+      </section>
+    </div>
+  );
+}
+
+// WARNING/TODO: this should be handling steps, not tasks
+function StepItem({ task, taskKey }) {
+  if (!task || !taskKey) return null;
+
+  const text = task.instruction;
+
+  return (
+    <li className="step-item">
+      <div className="grip-bar flex-row spacing-bottom-XS">
+        <GripIcon color="#A6A7A9" />
+      </div>
+      <ul className="tasks-list">
+        <li className="task-item">
+          <div className="flex-row spacing-bottom-M">
+            <span className="task-key">{taskKey}</span>
+            <span className="task-icon">
+              {/* TODO: change icon and aria label depending on task type*/}
+              <i
+                aria-label="Task type: text"
+                className="fa fa fa-file-text-o fa-fw"
+              />
```

Comment:
I don't think `<i>` can be labelled. Only elements with roles, like images, can be labelled. 

Does VoiceOver read this label? Does it show up in the accessibility tree?

Reply:
Font Awesome recommends `<span>` nowadays. Spans can't be labelled either, so I'm not sure what they mean by 'Accessibility' here:

https://fontawesome.com/docs/web/add-icons/how-to

Reply:
I ran some tests, and on macOS + Chrome:
- mac VoiceOver seems to detect _"Task type: text"_ label
- The Accessibility panel sees this as a _generic_ element. (Hmm, thinking I should add role=img for this)

<img width="301" alt="image" src="https://github.com/zooniverse/Panoptes-Front-End/assets/13952701/acef9f89-fbdc-42cd-a947-dc37833344a5">

Noted that a `<span>` may make more sense.

Reply:
All said though, I'm not too fond of going back to FontAwesome - I dislike how they're gating a lot of icons behind a subscription.

Reply:
Testing update:
- `<i aria-label="Task type: text" className="fa fa-whatever" />` and `<span aria-label="Task type: text" className="fa fa-whatever" />` appear to be functionally the same.
  - Oh, and one thing I missed - the voiceover calls it "Task type: text, EMPTY GROUP"
- Adding `role="img"` sets this to "Task type: text, image"
- Adding `role="presentation"` causes the task key and task question to be read together, e.g. "T0 This is the first task"... which actually sounds good, but misses out on the task type information.

Reply:
Generic elements can't be labelled and won't be read out by JAWS or NVDA, the two biggest screen readers. I'm pretty sure that's in the ARIA spec, so VO is ignoring the standard here. 

This relates to what I was saying about lists. Basically, labels are valid on elements that have a role.

> Because the generic role is nameless, the aria-labelledby and aria-label attributes are prohibited. Because the role is generic, the aria-roledescription and aria-brailleroledescription attributes are also prohibited.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/roles/generic_role